sql/branch.sql

```diff
@@ -0,0 +1,5 @@
+BEGIN;
+
+    ALTER TABLE participants DROP COLUMN goal;
```

Comment:
Okay, I see what you're doing - only deleting the columns that we won't find useful at all.